R/main.R

```diff
@@ -29,17 +29,18 @@ collection <- function(collection_name, connection, column_names = character(),
                        post_process = I) {
   make_sure_table_exists(connection, collection_name)
 
-  query <- post_process(rethinker::r()$db(DEFAULT_DB)$table(collection_name))
+  db_name <- connection$db_name
+  query <- post_process(rethinker::r()$db(db_name)$table(collection_name))
```

Comment:
`rethinker::r()$db(db_name)$table(collection_name)` seems to be a common operation, how about extracting this logic to some function like `get_table_handle(connection, collection_name)` ?